### ISSUE TITLE
Role term capitalization

### ISSUE THREAD

**Issue by justinlittman** [sul-dlss/dor-services-app#1742]
For druid:md128fd0077:

```
Original node:
<name type="corporate" usage="primary" authority="naf" authorityURI="http://id.loc.gov/authorities/names/" valueURI="http://id.loc.gov/authorities/names/no99051791.html">
  <namePart>Phoenicia, Arados</namePart>
  <role>
    <roleTerm type="text">state</roleTerm>
  </role>
</name>
Best guess roundtripped node:
<name type="corporate" valueURI="http://id.loc.gov/authorities/names/no99051791.html" authority="naf" authorityURI="http://id.loc.gov/authorities/names/" usage="primary">
  <namePart>Phoenicia, Arados</namePart>
  <role>
    <roleTerm type="text">State</roleTerm>
  </role>
</name>
```

Notice the changed capitalization. Perhaps a similar change needs to happen in normalization?

**Comment by jcoyne** [comment]
I think the problem here is that the normalizer does the downcasing.  Perhaps it shouldn't?

**Comment by justinlittman** [comment]
Looks like there are some cases in which it is downcased:
https://github.com/sul-dlss/dor-services-app/blob/master/app/services/cocina/from_fedora/descriptive/name_builder.rb#L220-L223

Perhaps the normalized needs to be more nuanced?

**Comment by arcadiafalcone** [comment]
Closed by #2081
